test/tests/searchTest.js

```diff
@@ -83,4 +83,58 @@ describe("Zotero.Search", function() {
 			assert.propertyVal(conditions[0], 'value', 'foo')
 		});
 	});
+
+	describe("#search()", function () {
+		let win;
+	
+		before(function () {
+			// Hidden browser, which requires a browser window, needed for charset detection
+			// (until we figure out a better way)
+			if (!Zotero.isStandalone) {
```

Comment:
This was apparently my code in attachmentsTest.js, but I have zero memory of this, and I can't imagine why it'd be necessary to test for Standalone every time we call loadBrowserWindow(). If we get tests running in Standalone, loadBrowserWindow() can just return the current window. So can take this out.